com.github.Xenoveritas.abuse.appdata.xml

```diff
@@ -22,10 +22,10 @@
   </releases>
   <screenshots>
     <screenshot type="default">
-      <image>http://abuse.zoy.org/raw-attachment/wiki/screenshots/title.png</image>
+      <image>https://raw.githubusercontent.com/flathub/com.github.Xenoveritas.abuse/refs/heads/master/screenshots/1.png</image>
     </screenshot>
     <screenshot>
-      <image>http://abuse.zoy.org/raw-attachment/wiki/screenshots/abuse.png</image>
+      <image>https://raw.githubusercontent.com/flathub/com.github.Xenoveritas.abuse/refs/heads/master/screenshots/2.png</image>
```

Comment:
Might be better to put these onto a CDN instead.

Reply:
I would not worry about that. Most AppStream parsers/UIs do cache screenshots instead of fetching them over and over again.

And there are many other Flatpaks and other projects directly linking the screenshots from GitHub in their AppStream metadata without any issues.

Also, what CDN would you recommend using?

Reply:
I usually submit at GitHub/Lab pages, but that is not an option here. Maybe https://screenshots.debian.net/package/abuse

Reply:
You could also upload them to https://github.com/AsciiWolf/asciiwolf.github.io

Reply:
I forgot about screenshots.debian.net. I will upload them there.

Thanks!